lib/tests-api.js

```diff
@@ -4,40 +4,26 @@ var Suite = require('./suite'),
 
 function nothing() {}
 
-function makeSkipBrowser(browser) {
-    if (typeof browser === 'string') {
-        return {browserName: browser};
+function buildBrowserSkipMatcher(browserToSkip, comment) {
```

Comment:
I want to check the `browser` object type and `throw new TypeError('suite.skip browser must be string or RegExp object')`. Should I put this check outside of this function?